core/src/test/java/org/infinispan/commands/GetAllCommandNodeCrashTest.java

```diff
@@ -41,6 +43,7 @@ public void test() throws Exception {
       cache(2).put(key, "value");
 
       ControlledRpcManager rpcManager = ControlledRpcManager.replaceRpcManager(cache(2));
+      rpcManager.excludeCommands(StateResponseCommand.class, StateTransferStartCommand.class);
```

Comment:
It's so weird that it wasn't a problem before, I have to debug the test to see how it's passing on master :)

Reply:
Yeah, I agree. But I wasn't quite sure what was going on.